src/ontology/obi-edit.owl

```diff
@@ -3590,6 +3590,10 @@ objectives is a planned process.</obo:IAO_0000116>
                         <owl:onProperty rdf:resource="http://purl.obolibrary.org/obo/OBI_0000312"/>
                         <owl:someValuesFrom rdf:resource="http://purl.obolibrary.org/obo/OBI_0000094"/>
                     </owl:Restriction>
+                    <owl:Restriction>
```

Comment:
Something is very wrong with this.: 'processed martial' has been modified to be equivalent to ''material entity' and (is_specified_output_of some 'material processing') and ('has role' some 'spike-in quality control role')'.

Reply:
Hey @DanBerrios. Thanks for working on these terms.

The change to the logical definition of 'processed material' must be unintended, but I can't tell for sure what the intended change is. Do you want to define 'ERCC RNA spike-in' as subclass of `'processed material'  and ('has role' some 'spike-in quality control role')`?

I can't merge this PR until that's figured out.

Reply:
@jamesaoverton you are correct: (1) the change to processed material was unintended and (2) 'ERCC RNA spike-in' is meant to be a subclass of 'processed material' and ('has role' some 'spike-in quality control role'). Note that 'SIRV RNA spike-in' should also be a subclass of 'processed material' and have the same role.